handlers/_swagger.js

```diff
@@ -171,6 +186,51 @@ function containsMultipartFormData(api) {
 
     return foundMultipartData;
 }
+/**
+ * The current use of this function is to figure out whether body parser needs to be configured
+ * This is the case when there is an operation that consumes form data without a file
+ * If there is a file, then multer will be used to parse form data
+ * @param api
+ * @returns {boolean}
+ */
+function containsUrlEncodedFormData(api) {
+    if (_.indexOf(api.consumes, 'application/x-www-form-urlencoded') > -1) {
```

Comment:
What would be the effect on our algorithm if there is a top-level `consumes` of `application/x-www-form-urlencoded`, and a particular method that overrides it to unset it?

We may need to carry the global value, if it exists, through the loop that is checking all paths.

**from the OpenAPI spec**

global `consumes`:
> A list of MIME types the APIs can consume. This is global to all APIs but can be overridden on specific API calls. Value MUST be as described under Mime Types.

path-specific `consumes`:
> A list of MIME types the operation can consume. This overrides the consumes definition at the Swagger Object. An empty value MAY be used to clear the global definition. Value MUST be as described under Mime Types.